packages/backend-dev-utils/src/ipcClient.ts

```diff
@@ -61,6 +68,11 @@ export class BackstageIpcClient {
 
   constructor(sendMessage: SendMessage) {
     this.#sendMessage = sendMessage;
+    process.addListener('message', (message: unknown, _sendHandle: unknown) => {
+      if (isResponse(message)) {
```

Comment:
Tbh I think it's cleaner to inline the `isResponse` check straight into the listener here, rather than keeping it separate.

Reply:
Totally inlining this would require also type check for the `type` so added the `isResponse` inside the `handleMessage` block. So instead of writing:

```ts
if(typeof (msg as Response)?.type === 'string' && (msg as Response)?.type === responseType)) {}
```

it can be done with

```ts
const isResponse = (msg: unknown): msg is Response =>
  (msg as Response)?.type === responseType;
if(isResponse(message)) {}
```